content/03.results.md

```diff
@@ -39,10 +39,15 @@ More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/ne
 In 2011, the Children’s Brain Tumor Tissue Consortium, now known as the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection.
 Since the CBTN opened its protocol before molecular data were integrated into classifications, the majority of the samples within the OpenPBTA lacked molecular subtype annotations.
 Moreover, the OpenPBTA data does not feature methylation arrays which are now commonly used to perform molecular subtyping.
-Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (**Table S1**).
+Therefore, we jointly considered key genomic features of tumor entities described by the WHO in 2016, low-grade glioma (LGG) subtypes described by Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], as well as clinician and pathologist review to subtype 60% (641/1074) of tumor samples with high confidence (**Table S1**). <!--SAMPLECOUNT-->
+<!--QUESTION: Do we want to use the "integrated" count here or the number subtyped.
+I have used the molecular subtyped count, but the earlier part of the sentence discussed WHO, and that is what is in the summary, so I wasn't sure.
+For reference, the WHO integrated # is 630 or 59%-->
+
 Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR incorrectly entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using current WHO terms (e.g., tumors formerly ascribed primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
-**Table {@tbl:Table1}** lists the subtypes we defined within the OpenPBTA, comprising of LGATs (low grade astrocytic tumors; N = 569), high-grade gliomas (N = 314), embryonal tumors (N = 229), ependymomas (N = 65), tumors of sellar region (N = 51), mesenchymal non-meningothelial tumors (N = 21), glialneuronal tumors (N = 20), and chordomas (N = 12).
-For detailed methods, see **STAR Methods** and **Figure {@fig:S1}**. 
+<!--TODO: Update counts below when table is updated-->
```

Comment:
This is merged now, but maybe we should put this in a new PR - do you want to update this or should I?

Reply:
I was expecting this to be a separate PR. I am agnostic about who does it!